addons/dynamishelper/dynamishelper.lua

```diff
@@ -24,408 +24,597 @@
+help = T{
+size = "Usage: dh size [font size] - Your current size is %u pixel",
+font = "Usage: dh font [font name] - You are currently using %q",
+opacity = "Usage: dh opacity [0-100]%% - Opacity is currently at %u%%",
+padding = "Usage: dh padding [size] - Padding is currently %u pixels",
+bgcolor = "Usage: dh bgcolor [Red] [Green] [Blue] - Background color - Example: 255 0 0 is red",
+stroke = "Usage: dh stroke [width] - Stroke width is currently %u pixels",
+stopacity = "Usage: dh stopacity [0-100]%% - Stroke opacity is currently at %u%%",
+stcolor = "Usage: dh stcolor [Red] [Green] [Blue] - Stroke color - Example: 255 0 0 is red",
+posx = "Usage: dh posx [x] - Current window position is x=%u",
+posy = "Usage: dh posy [y] - Current window position is y=%u",
+ll = "Usage: dh ll create: Creates a light luggage profile to lot all dynamis currency.'"
+}
+
+function printHelp(command,val)
+    if not command and not val then
+        windower.add_to_chat(159,'\nDynamisHelper v2.0')
+        windower.add_to_chat(159,'dh size [number]: Change the font size.')
+        windower.add_to_chat(159,'dh font [Arial, Tahoma, Times "Open Sans" ...]: Change the font.')
+        windower.add_to_chat(159,'dh posx [pixel]: Position on the X axis in pixel.')
+        windower.add_to_chat(159,'dh posy [pixel]: Position on the Y axis in pixel.')
+        windower.add_to_chat(159,'dh bgcolor [Red] [Green] [Blue] - Background color - Example: 255 0 0 is red')
+        windower.add_to_chat(159,'dh opacity bg_alpha: Opacity (0-255) of the background.')
+        windower.add_to_chat(159,'dh visible: Toggle addon window.')
+        windower.add_to_chat(159,'dh bold: Toggle bold text.')
+        windower.add_to_chat(159,'dh padding [size]: Padding of the text window.')
+        windower.add_to_chat(159,'dh stroke [width]: Stroke width of the text.')
+        windower.add_to_chat(159,'dh stopacity [0-100]%: Stroke opacity.')
+        windower.add_to_chat(159,'dh stcolor [Red] [Green] [Blue] - Stroke color - Example: 255 0 0 is red')
+
+        -- compatibility commands ---------------------------------------------
+        windower.add_to_chat(159,'dh ll create: Creates a light luggage profile to lot all dynamis currency.')
+        -----------------------------------------------------------------------
+        windower.add_to_chat(159,'dh save: save your current settings.')
+    else
+        local m = string.format(help[command],val)
+        windower.add_to_chat(159,'\nDynamisHelper v2.0')
+        windower.add_to_chat(159,m)
+    end
 end
 
-windower.register_event('unload',function ()
- 	windower.text.delete('dynamis_box')
- 	windower.text.delete('proc_box')
+-------------------------------------------------------------------------------
+-- Process options and save settings ------------------------------------------
+-------------------------------------------------------------------------------
+windower.register_event('addon command',function (...)
```

Comment:
Instead of using `:lower()` dozens of times below, you can use it just once at the top by designing the function with a more fitting signature:
```lua
windower.register_event('addon command', function(command, ...)
    command = command and command:lower() or 'help'
    local options = {...}
```

This will replace the first 7 lines of the function, and you can remove all `:lower()` calls on command below